app/controllers/submit.js

```diff
@@ -233,6 +234,11 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
     doiValid: alias('validations.attrs.basicsDOI.isValid'),
     originalPublicationDateValid: alias('validations.attrs.basicsOriginalPublicationDate.isValid'),
 
+    // Sloan waffles
+    sloanCoiDisplayEnabled: alias('features.sloanCoiDisplay'),
```

Comment:
I maybe wrong but I think the flags (switches rather) you want to check are `sloanCoiInput`, `sloanDataInput`, `sloanPreregInput`